src/Symfony/Component/ObjectMapper/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/object-mapper",
+    "type": "library",
+    "description": "Object to object mapper",
```

Comment:
Let's do something similar to other components like `Provides a way to map an object to another object`.